doc/asciidoc/producer/configuration.adoc

```diff
@@ -15,6 +15,7 @@ kafka.connection.timeout.ms=10000
 kafka.replication=1
 kafka.linger.ms=1
 kafka.transactional.id=
+kafka.topic.discovery.polling.interval=300000
```

Comment:
This is a good start but request you add another bit of docs to a different section specifying how the plugin handles non-existant topics.  From reading the PR, the answer I guess is that it discards messages to non-existant topics, and warns at startup about this problem that it detects.  This polling interval controls how often we re-check for topics, so it can pick up on new topics created since the server started.

We just need to note this in the documentation, since otherwise to discover this plugin behavior you have to already know what the configuration item is, which people won't.

Reply:
I extended the documentation with this:

_In case you Kafka broker is configured with `auto.create.topics.enable` to `false`,
all the messages sent to topics that don't exist are discarded;
this because the `KafkaProducer.send()` method blocks the execution, as explained in [KAFKA-3539](https://issues.apache.org/jira/browse/KAFKA-3539).
You can tune the custom property `kafka.topic.discovery.polling.interval` in order to
periodically check for new topics into the Kafka cluster so the plugin will be able
to send events to the defined topics._

Reply:
@conker84 good to go as far as mergeability but I see the CI system is broken, so please investigate.  I'd merge on green build.

Reply:
Something went wrong with Travis, I restarted it 3 times and it worked. Now it's green